packages/teleport-component-generator-react/src/react-component.ts

```diff
@@ -1,9 +1,10 @@
 import { AssemblyLine, Builder, Resolver, Validator } from '@teleporthq/teleport-generator-core'
 
 import reactComponentPlugin from '@teleporthq/teleport-plugin-react-base-component'
-import reactInlineStylesPlugin from '../../teleport-plugin-react-inline-styles/lib'
-import reactPropTypesPlugin from '../../teleport-plugin-react-proptypes/lib'
-import importStatementsPlugin from '../../teleport-plugin-import-statements/lib'
+import reactInlineStylesPlugin from '@teleporthq/teleport-plugin-react-inline-styles'
+import reactStyledJSXPlugin from '@teleporthq/teleport-plugin-react-styled-jsx'
+import reactPropTypesPlugin from '@teleporthq/teleport-plugin-react-proptypes'
```

Comment:
@alexnm if we bundle this way, we may end up packing all the supported plugins again in the component-generator right? which in returns effects the bundle size, in that way we might lose the power of splitting the plugins into submodules. 🤔

Reply:
it's a temporary solution, the idea is to have feature parity with the old style monorepo, then we start working on dynamic plugins